Scripts/prepare_modes.py

```diff
@@ -279,9 +279,9 @@ def prepare_filter_mode_smart(scanMode, config, miscData, sensitive=False):
     pass
   else:
     if sensitive:
-      print("\n----------------------------------------------- Sensitive-Smart Mode -----------------------------------------------")
-    else: # if not sensitive
-      print("\n----------------------------------------------- Auto-Smart Mode -----------------------------------------------")
+      print(f"\n----------------------------------------------- {F.YELLOW}Sensitive-Smart Mode{S.R} -----------------------------------------------")
+    else: # if not sensitive	    else: # if not sensitive
```

Comment:
```suggestion
    else: # if not sensitive
```

Not sure where that duplicated code came from, even if it would be a comment.